inc/thorvg.h

```diff
@@ -1135,6 +1145,13 @@ class TVG_API Shape final : public Paint
      */
     StrokeJoin strokeJoin() const noexcept;
 
+    /**
+     * @brief Gets the stroke miterlimit.
+     *
+     * @return The stroke miterlimit value when succeed, -1 if no stroke was set.
```

Comment:
Conceptually, there shouldn't be a false scenario in a stroke. 
I believe it would be better to return the default value (4) rather than -1.